### ISSUE TITLE
Problems in reproducing the results

### ISSUE THREAD

**Issue by RiHY99** [caicch/ICT-Net#4]
Hi! Thanks for your work and code.
But after running your code, I find that I can not reproduce the results in your paper. I did not change any settings in your code, and the following are the results I obtained when setting the beam size to 1 and 3, respectively:

beam size 1: BLEU-1 0.8365782043877742 BLEU-2 0.7464677749906835 BLEU-3 0.6737276908055432 BLEU-4 0.6160959855227558 METEOR 0.38670208813735063 ROUGE_L 0.7318392543520593 CIDEr 1.3008293088859515

beam size 3: BLEU-1 0.8393105603275597 BLEU-2 0.7571647725715319 BLEU-3 0.6914677794434455 BLEU-4 0.6407943985201704 METEOR 0.38451187006310084 ROUGE_L 0.7314299574060167 CIDEr 1.2952080545750886

I want to know if I made any errors while running the code and how to get the results in your paper? Or is this result acceptable?

**Comment by caicch** [comment]
Hi, thank you for your interest in our work. For this work, the result may vary due to the seed, we choose the best model instead. You could set a seed, change batch size, learning rate, etc, to find your best performance model.

**Comment by RiHY99** [comment]
Thanks for your quick reply! I noticed that in another issue https://github.com/caicch/ICT-Net/issues/3, you said you did not change any parameters when you uploaded the code. But I can not reproduce the results with the setting in your code. I wonder if this result is acceptable or not, and if it is not acceptable , can you provide the seed and other settings in your experiment?

**Comment by caicch** [comment]
The result seems acceptable, since it performs better than baseline models in many metrics. But you can monitor the result for every epoch of training and find the model with better performance. As you probably observed, this code is not that stable, following the existing referenced method. I uploaded the parameters that I used to achieve good performance on my machine. You can adjust them to find yours.

**Comment by RiHY99** [comment]
Thanks for your quick reply.
